File: src/main/java/com/itechart/tarasevi/logic/domain/ContactPhone.java
package com.itechart.tarasevi.logic.domain;

import java.io.Serializable;

public class ContactPhone implements Serializable{

    private int id;
    private int employeeID;
    private int codeCountry;
    private int codeOperator;
    private int number;
    private String type;
    private String comment;

    public int getId() {
        return this.id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getEmployeeID() {
        return employeeID;
    }

    public void setEmployeeID(int employeeID) {
        this.employeeID = employeeID;
    }

    public int getCodeCountry() {
        return this.codeCountry;
    }

    public void setCodeCountry(int codeCountry) {
        this.codeCountry = codeCountry;
    }

    public int getCodeOperator() {
        return this.codeOperator;
    }

    public void setCodeOperator(int codeOperator) {
        this.codeOperator = codeOperator;
    }

    public int getNumber() {
        return this.number;
    }

    public void setNumber(int number) {
        this.number = number;
    }

    public String getType() {
        return this.type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getComment() {
        return this.comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    @Override
    public String toString() {
        return "ContactPhone{" +
                "id=" + id +
                ", employeeID=" + employeeID +
                ", codeCountry=" + codeCountry +
                ", codeOperator=" + codeOperator +
                ", number=" + number +
                ", type='" + type + '\'' +
                ", comment='" + comment + '\'' +
                '}';
    }
}

File: src/main/webapp/web/js/entity/contactPhone.js
function ContactPhone() {
    this._id = undefined;
    this._rowIndex = undefined;
    this._codeCountry = undefined;
    this._codeOperator = undefined;
    this._number = undefined;
    this._type = undefined;
    this._comment = undefined;
}
ContactPhone.prototype = {
    set id(id){
        this._id = id;
    },
    get id(){
        return this._id;
    },
    set rowIndex(rowIndex) {
        this._rowIndex = rowIndex;
    },
    get rowIndex() {
        return this._rowIndex;
    },
    set codeCountry(codeCountry) {
        this._codeCountry = codeCountry;
    },
    get codeCountry() {
        return this._codeCountry;
    },
    set codeOperator(codeOperator) {
        this._codeOperator = codeOperator;
    },
    get codeOperator() {
        return this._codeOperator;
    },
    set number(number) {
        this._number = number;
    },
    get number() {
        return this._number;
    },
    set type(type) {
        this._type = type;
    },
    get type() {
        return this._type;
    },
    set comment(comment) {
        this._comment = comment;
    },
    get comment() {
        return this._comment;
    }
};
